source/MultiGrid/MultiGrid.js

```diff
@@ -47,6 +49,8 @@ export default class MultiGrid extends React.PureComponent {
     styleBottomRightGrid: {},
     styleTopLeftGrid: {},
     styleTopRightGrid: {},
+    hideTopRightGridScrollbar: false,
```

Comment:
nit: I don't think there is a need to specify defaultProps for these. Since it would be `undefined` if not passed in. Which is a falsey value

Reply:
I added because there was an existing pattern for `enableFixedColumnScroll` and `enableFixedRowScroll`. So followed it.